unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/MenuSections/SectionLandView.cs

```diff
@@ -5,14 +5,20 @@
 
 internal class SectionLandView : MonoBehaviour, IDisposable
 {
+    public event Action OnOpenMarketplaceRequested;
+
     [SerializeField] public LandElementView landElementView;
     [SerializeField] public ScrollRect scrollRect;
     [SerializeField] public GameObject contentContainer;
     [SerializeField] public GameObject emptyContainer;
+    [SerializeField] public GameObject loadingAnimationContainer;
+    [SerializeField] public GameObject noSearchResultContainer;
     [SerializeField] public Button buttonNoLandCTA;
```

Comment:
Maybe we should remove `[SerializeField]` or make the fields private. `[SerializeField]` with public members is redundant.

Reply:
Made them private and/or internal.
I know that `[SerializeField]` with `public` is redundant, but I like to use `[SerializeField]` like a "tag" telling me "this stuff is set in the gameObject and not by code" 😅
In any case, there were no reason to keep those fields as public. Thanks